src/components/shop/shop-menu-screen.js

```diff
@@ -10,10 +10,12 @@ import { onScaleChange } from "../../core/scaler.js";
 import { createTitles } from "../../core/titles.js";
 import { createBalance } from "./balance-ui.js";
 import { createMenu } from "./menu.js";
+import { initResizers } from "./backgrounds.js";
 
 export class ShopMenu extends Screen {
     preload() {
         this.plugins.installScenePlugin("rexUI", RexUIPlugin, "rexUI", this, true);
+        initResizers();
```

Comment:
can you init the resizers as part of `addBackgroundItems`, or would that be polluting it with Rex?

Reply:
Sadly that is called for every scene - not just ones with the ui plugin included.